warning/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for warning 3.0.0
+// Project: https://github.com/BerkeleyTrue/warning
+// Definitions by: Chi Vinh Le <https://github.com/cvle>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const warning: (condition: any, format?: string, ...extra: any[]) => void;
```

Comment:
You can just `declare function warning(condition: any, format?: string, ...extra: any[]): void;`.